app/src/main/java/org/simple/clinic/settings/changelanguage/ChangeLanguageScreen.kt

```diff
@@ -55,14 +55,13 @@ class ChangeLanguageScreen(
   private val uiRenderer = ChangeLanguageUiRenderer(this)
 
   private val delegate: MobiusDelegate<ChangeLanguageModel, ChangeLanguageEvent, ChangeLanguageEffect> by unsafeLazy {
-    MobiusDelegate(
+    MobiusDelegate.forView(
         events = events,
         defaultModel = ChangeLanguageModel.FETCHING_LANGUAGES,
         init = ChangeLanguageInit(),
         update = ChangeLanguageUpdate(),
         effectHandler = effectHandler.create(uiActions = this).build(),
-        modelUpdateListener = uiRenderer::render,
-        crashReporter = crashReporter
```

Comment:
I guess we can remove this `crashReporter` injection from the class as well now.